package.json

```diff
@@ -2,18 +2,21 @@
   "name": "vite-plugin-svelte-monorepo",
   "private": true,
   "scripts": {
-    "dev": "pnpm --dir packages/vite-plugin-svelte dev",
-    "build": "pnpm --dir packages/vite-plugin-svelte build",
+    "build": "echo 'no need to build'",
```

Comment:
It would be nice to remove this entirely since the npm/pnpm runner should also report something similar.

Reply:
done